public/posts/v1.8.md

```diff
@@ -411,6 +451,10 @@ Results in:
 You can run this script yourself:
 `deno run --unstable https://deno.land/posts/v1.8/emit_iife.ts`.
 
+This is particularly useful for creating bundles for older browsers that do not
+support ESM. For browsers that do support ESM, these bundles should be used aas
+they are generally smaller.
```

Comment:
I'm not sure if last sentence is true... also `aas` should be `as`?

Reply:
Mh - I know this is true for webpack / parcel and friends, but indeed not sure about our bundler. I will remove that line.